app/src/main/java/hous/release/android/presentation/login/DatePickerDialog.kt

```diff
@@ -0,0 +1,78 @@
+package hous.release.android.presentation.login
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import hous.release.android.R
+import hous.release.android.databinding.DialogDatePickerBinding
+import hous.release.android.util.extension.initLayout
+import java.lang.IllegalStateException
+
+class DatePickerDialog(private val doAfterConfirm: (date: String) -> Unit) :
+    DialogFragment() {
+    private var _binding: DialogDatePickerBinding? = null
+    val binding get() = _binding ?: error(getString(R.string.binding_error))
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        _binding = DialogDatePickerBinding.inflate(requireActivity().layoutInflater)
+        initConfirmTextClickListener()
+        initCancelTextClickListener()
+        return activity?.let {
+            val dialog = AlertDialog.Builder(it).create()
+            dialog.setView(binding.root)
+            dialog
+        } ?: throw IllegalStateException()
+    }
+
+    override fun onStart() {
+        super.onStart()
+        initLayout()
+    }
+
+    private fun sendSelectedDate() {
+        with(binding.datePickerDialogDatePicker) {
+            val yearFormat = getYearFormat(year)
+            val monthFormat = getMonthFormat(month)
+            val dayFormat = getDayFormat(dayOfMonth)
+            val date = "$yearFormat-$monthFormat-$dayFormat"
+            doAfterConfirm(date)
+        }
+        dismiss()
+    }
+
+    private fun getYearFormat(year: Int): String = year.toString()
+
+    private fun getMonthFormat(month: Int): String {
+        return if (month < 9) {
+            "0" + (month + 1).toString()
+        } else {
+            (month + 1).toString()
+        }
+    }
```

Comment:
```suggestion
    private fun getMonthFormat(month: Int): String = "%02d".format(month+1)
```
format을 사용해서 코드를 더 줄일 수 있을 거 같습니다

Reply:
영어로 세팅한 사람들도 있을 것 같아 디자인 수정하였습니다
(년/월/일 선 길이가 일정해짐)